src/net.cpp

```diff
@@ -458,13 +458,30 @@ bool CNode::IsBanned(CNetAddr ip)
     return fResult;
 }
 
-bool CNode::Ban(const CNetAddr &addr) {
+bool CNode::Ban(const CNetAddr &addr, int64_t bantimeoffset) {
```

Comment:
Understood, but IMHO a function that doesn't need it could just be ``void``. Perhaps you can just add a commit for that at the end? At least it could be changed for ``GetBanned`` as you just added it :).